shared/middleware/animation_library_api.rb

```diff
@@ -43,10 +43,11 @@ class AnimationLibraryApi < Sinatra::Base
   # Retrieve the manifest from S3. This will be extended to take in
```

Comment:
Do you mind updating this comment? I'm still planning on adding locale to this function but we can remove the gamelab support piece.